landoui/app.py

```diff
@@ -9,6 +9,10 @@
 from flask_assets import Environment
 from webassets.loaders import YAMLLoader
 
+from landoui import auth
+
+oidc = None
```

Comment:
Please add a comment about why this var has to be global (for the time being).

I don't like having a naked global variable here.  We should talk about refactoring this next sprint: either push the var into the application config, or plugging the functionality into the framework.